apps/dokploy/server/utils/docker/compose/network.ts

```diff
@@ -20,15 +25,20 @@ export const addPrefixToServiceNetworks = (
 		if (service.networks) {
 			// 1 Case the most common
 			if (Array.isArray(service.networks)) {
-				service.networks = service.networks.map(
-					(network: string) => `${network}-${prefix}`,
-				);
+				service.networks = service.networks.map((network: string) => {
+					if (network === "dokploy-network") {
+						return "dokploy-network";
+					}
+					return `${network}-${prefix}`;
+				});
 			} else {
 				// 2 Case
-				service.networks = _.mapKeys(
-					service.networks,
-					(_value, key) => `${key}-${prefix}`,
-				);
+				service.networks = _.mapKeys(service.networks, (_value, key) => {
+					if (key === "dokploy-network") {
+						return "dokploy-network";
+					}
+					return `${key}-${prefix}`;
+				});
```

Comment:
This is tiny, not a real issue, but if we use thing like `-this`, it should be called `suffix` :P

Reply:
Thank you for pointing out!